src/assignments/hello-world.rst

```diff
@@ -1,4 +1,65 @@
+
+Now, turn your attention to ``Program.cs`` in the ``HelloWorldAutograded`` project. We have given you some code that will output the string returned by the ``SayHello()`` function in ``SayHelloClass``.
+
+To run the program, hit the *Run* button at the top right-hand corner of the window or right-click on the ``HelloWorldAutograded`` project and select *Run Project*.
+When the program runs, Visual Studio will open a pane at the bottom of the window and display program output. Currently, the program has no output. You'll fix that in a bit.
+
+Running the Autograding Script
+------------------------------
+
+For this unit, we encourage you to run the autograding tests yourself *before* you push your code.
+This will make it quicker and easier to know whether or not your code is correct. Let's learn how to run the tests.
+
+In the *Solution Explorer*, navigate to the test class, ``AutogradingTests``, in the project ``HelloWorldTests``.
+For each assignment, there will be a test project, which contains all of the autograding tests and may have more than one class.
+
+A test is a single method in a test file that has ``[TestMethod]`` just above it. A test file may have multiple tests. To run just one test, you can right-click on the test method name and select *Run Test*.
+To run all of the auto-grading tests (in this case, there is only one) but right-clicking on the project name and selecting *Run Project*.
+
+When Visual Studio runs the tests, it opens a pane at the bottom of the window to display test results. At the left, we see that the test has failed. We can expand the tree to see each of the individual tests (again, in this case there is only one). Selecting one of the tests shows its results to the right. 
+
+In this case, the test expected our code to output ``Hello, World!``. Instead, it provided no output at all.
+
+.. admonition:: Warning
+
+   The autograding tests are VERY exacting.
+   A difference of just one character will result in a failed test.
+   The tests are also case-sensitive. You'll need to pay attention to detail in order complete your assignments.
+
+When your code is correct, Visual Studio will display a green checkmark indicating passing tests.
+
+Your Task
+---------
+
+Your task is simple: make the program print out the string ``"Hello, World!"``.
+Edit the code in the ``SayHelloClass`` class, within the ``SayHello()`` method. When your code is correct, running the tests will display passing results.
+
+.. _submitting-your-work:
+
+Submitting Your Work
+--------------------
+
+Once you get all of the tests to pass, open up Visual Studio's *Terminal* pane. Then you should commit and push your code to GitHub.
+
+Visit your assignment repository page. Near the top right, you'll see a green checkmark indicating that GitHub has graded your assignment as passing. If you see a red X, then your assignment is not yet correct.
```

Comment:
GitHub Classroom?

Reply:
This actually takes place on the standard Github site too, so I think it makes more sense with just Github